Misc/NEWS.d/next/Library/2019-07-31-15-52-51.bpo-35943.-KswoB.rst

```diff
@@ -0,0 +1 @@
+Added optimization to prevent `PyImport_GetModule()` from returning partially-initialized module.
```

Comment:
Oh also:

This NEWS entry (and the commit message) say "optimization", but the change is actually better than that! This changes the function's behavior, in a way that makes the API safer and makes it easier to avoid bugs when using it. So let's write the description to make that clear. :slightly_smiling_face: 

One good description might be:
```
The function :c:func:`PyImport_GetModule` now ensures any module it returns is fully initialized.
```

Also I think `C API` rather than `Library` would be the most helpful home for the NEWS entry.